src/Resources/contao/config/config.php

```diff
@@ -8,4 +8,10 @@
  * @license http://www.gnu.org/licences/lgpl-3.0.html LGPL
  */
 
-$GLOBALS['TL_HOOKS']['getAttributesFromDca']['flatpickr'] = [\HeimrichHannot\FlatpickrBundle\EventListener\GetAttributesFromDcaListener::class, 'onGetAttributesFromDca'];
\ No newline at end of file
+$GLOBALS['TL_HOOKS']['getAttributesFromDca']['huh_flatpickr'] = [
```

Comment:
Hui, einen Key so umzubenennen ist eigentlich ein Breaking Change ;) Aber ich würde es in dem Fall erstmal lassen. Kann man Rückgängig machen, wenn sich jemand beschwert.

Reply:
Dann lieber nicht

Reply:
ok :D